jmespath-core/src/main/java/io/burt/jmespath/BaseAdapter.java

```diff
@@ -1,12 +1,11 @@
 package io.burt.jmespath;
 
-import java.util.List;
+import java.util.Comparator;
 import java.util.Iterator;
+import java.util.List;
 
-import io.burt.jmespath.function.FunctionRegistry;
 import io.burt.jmespath.function.ExpressionOrValue;
-
-import static io.burt.jmespath.JmesPathType.*;
+import io.burt.jmespath.function.FunctionRegistry;
```

Comment:
Please don't let Eclipse do random changes like moving imports for no reason (`List` was moved above too).